lib/LedgerSMB/Sysconfig.pm

```diff
@@ -163,7 +167,7 @@ def 'force_username_case',
 
 def 'max_post_size',
     section => 'main',
-    default => 1024 * 1024,
+    default => MAX_POST_SIZE_DEFAULT,
```

Comment:
@ziuq the problem I have with putting this constant declaration ANYWHERE else except in Sysconfig.pm it's self is quite simple.
The line `default => 4194304,` is, it's self, the declaration of that constant.

All of the default values in this file are constants, (and as such we probably should trye to make the declaration's behave as such.)

Declaring an intermediate constant does not in any way help readability nor maintainability of the codebase, in fact it hinders both.

As such I am extremely resistant to hiding this value behind a constant declaration.

@ehuelsmann needs to have the final say here, but I suspect he will agree that it should stay put.

I'm not certain, but if we can slightly alter the code here so that critic considers it a constant declaration (which it is) rather than use of a magic number, that would mean not needing "no-critic"